tests/utils.py

```diff
@@ -875,6 +876,72 @@ def make_field_data(monitor: td.FieldMonitor) -> td.FieldData:
             **field_cmps,
         )
 
+    def make_field_time_data(monitor: td.FieldTimeMonitor) -> td.FieldTimeData:
```

Comment:
@tylerflex Make sure to check these changes in case I missed something. I used these extensions to tests the different types of input data to the impedance calculator.

Reply:
just looked it over and seems good to me.